tools/prepublish.ts

```diff
@@ -37,44 +37,36 @@ const extract = (input, opts, callback) => {
       });
 };
 
-async.series(
-    [
-      (next) => {
-        require('child_process').exec('rm -rf google', next);
-      },
+async function main() {
+  await require('child_process').exec('rm -rf google');
 
-      (next) => {
-        extract(
-            'https://github.com/googleapis/googleapis/archive/master.zip', {
-              strip: 1,
-            },
-            next);
+  await extract(
```

Comment:
Are you sure that `extract` returns a promise?  It looks like you're just passing an empty closure as the callback.  You may need to use `util.promisify` here.